src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```
@@ -42,12 +50,26 @@ export class ActionBarOverflowMenu {
                 items={this.items}
                 onSelect={this.handleSelect}
             >
-                <button slot="trigger">{this.countOverflowedItems()}</button>
+                <button slot="trigger">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)